src/types/api.ts

```diff
@@ -0,0 +1,17 @@
+import type { Repository, Revision } from './state';
+
+type APIFilterParams = {
+  full: boolean;
```

Comment:
what does `full` stand for?

Reply:
> what does `full` stand for?

I think it's a query param used to make the call Perfherder's Compare View to get the recent pushes. @kimberlythegeek I've looked at the code and it seems that this param is not needed as per this comment https://github.com/mozilla/treeherder/blob/master/treeherder/webapp/api/push.py#L174